WEBSITE_CONTRIBUTING.md

```diff
@@ -11,6 +11,77 @@ Contributing to the Podman Desktop website is a valuable way to support the proj
 
 The below information outlines details on how to both contribute as well as our documentation "templates" and criteria.
 
+## Getting Started
+
+### Install the project dependencies
+
+```shell-session
+$ pnpm install
+```
+
+### Local Development of the website / documentation
+
+```shell-session
+$ pnpm start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+You may also run:
+
+```sh
+pnpm website:dev
+```
+
+Which will automatically open your browser to `localhost:3000`.
+
+### Build
+
+```shell-session
+$ pnpm build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+You may also build a "production-like" environment by using:
+
+```sh
+pnpm website:prod
+```
```

Comment:
duplicate information. Already covered [here](https://github.com/podman-desktop/podman-desktop/blob/main/WEBSITE_CONTRIBUTING.md#previewing-the-website).